docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -35,7 +35,7 @@ This procedure assumes familiarity with Docker and Docker Compose. If you haven'
 Follow these steps to install the necessary tools, if you have not already done so.
 
 1. Install `Docker Community Edition (CE) <https://docs.docker.com/engine/installation/>`__ on your workstation. Depending on your OS, you may need to configure Docker to use at least 4.00 GB of memory for the Airflow containers to run properly. Please refer to the Resources section in the `Docker for Windows <https://docs.docker.com/docker-for-windows/#resources>`__ or `Docker for Mac <https://docs.docker.com/docker-for-mac/#resources>`__ documentation for more information.
-2. Install `Docker Compose <https://docs.docker.com/compose/install/>`__ v1.29.1 or newer on your workstation.
+2. Install `Docker Compose <https://docs.docker.com/compose/install/>`__ v2.14.0 or newer on your workstation.
```

Comment:
I couldn't come up with anything better here rather than pick up current MIN_DOCKER_COMPOSE_VERSION for CI/Breeze 🤣 At least we run compose tests by use 2.14

Reply:
That's a very good one. I even explained that very minimum is coming in this PR today